gui/label.py

```diff
@@ -0,0 +1,18 @@
+import pygame
+
+
+class Label():
+    def __init__(self, x_pos, y_pos, text_input, font="comicsans", font_size=50, color ="yellow"):
```

Comment:
I'm wonder shouldn't we all use hex code as a basic way to colors. it's more flexible to pick right shade of color and unification of code will looks better.

Reply:
Yes, we didn't talk about this but that will be the best solution: colors added to config.ini with clear names (like: `background_gray` or sth) and hex code as value. Please remember to pick this topic on next meeting if I forget